config/config.schema.json

```diff
@@ -15,6 +15,19 @@
         "roleName"
       ]
     },
+    "tracing": {
+      "type": "object",
+      "properties": {
+        "address": {
+          "type": "string",
+          "description": "Defines exporter address"
+        },
+        "exporter": {
+          "type": "string",
+          "description": "Defines which exporter the SDK should use"
```

Comment:
I wonder if we really need to expose the exporter setting?

Reply:
I think last time we discussed that it may makes sense. Since multiple exporters like `console` are supported.